blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.xsl

```diff
@@ -21,6 +21,11 @@
             </xsl:call-template>
             </dt>
             <dd class="ff_module-form-preview__list__data">
+                <xsl:if test="@data-ff-preview-for">
```

Comment:
Ah, sorry, I wasn't very clear: I meant just `@preview-for` since we're already in the context of Firefly. Not too fussed about it though.